pkg/secrethub/credentials/sessions/session_aws.go

```diff
@@ -80,3 +83,14 @@ func getCallerIdentityRequest(region string, awsCfg ...*aws.Config) ([]byte, err
 	}
 	return buf.Bytes(), nil
 }
+
+func handleAWSErr(err error) error {
+	errAWS, ok := err.(awserr.Error)
+	if ok {
+		if errAWS.Code() == "NoCredentialProviders" {
+			return shaws.ErrNoAWSCredentials
+		}
+		err = errio.Namespace("aws").Code(errAWS.Code()).Error(errAWS.Message())
+	}
+	return err
+}
```

Comment:
Should we make the existing function in `internals/aws/errors.go` exported so we can use it here as well and don't have to dupliate it?

Reply:
I wasn't sure about exporting the function but it does seem like a better idea than duplicating code.